Mathlib/LinearAlgebra/FreeModule/PID.lean

```diff
@@ -289,18 +289,16 @@ theorem Submodule.nonempty_basis_of_pid {ι : Type*} [Finite ι] (b : Basis ι R
     (N : Submodule R M) : ∃ n : ℕ, Nonempty (Basis (Fin n) R N) := by
   haveI := Classical.decEq M
   cases nonempty_fintype ι
-  induction' N using inductionOnRank with N ih
-  · exact b
-  · let b' := (b.reindex (Fintype.equivFin ι)).map (LinearEquiv.ofTop _ rfl).symm
-    by_cases N_bot : N = ⊥
-    · subst N_bot
-      exact ⟨0, ⟨Basis.empty _⟩⟩
-    obtain ⟨y, -, a, hay, M', -, N', N'_le_N, -, -, ay_ortho, h'⟩ :=
-      Submodule.basis_of_pid_aux ⊤ N b' N_bot le_top
-    obtain ⟨n', ⟨bN'⟩⟩ := ih N' N'_le_N _ hay ay_ortho
-    obtain ⟨bN, _hbN⟩ := h' n' bN'
-    exact ⟨n' + 1, ⟨bN⟩⟩
-  infer_instance
+  induction N using inductionOnRank b with | ih N ih =>
```

Comment:
The diff is hard to read - but this is just providing `b`, removing the now-superfluous last `infer_instance` and un-indenting, right?

Reply:
Exactly, yeah. (Github has a :gear: icon at the top of the "Files changed" view that allows hiding whitespace changes)